frontend/app/routes/protected/person-case/birth-details.tsx

```diff
@@ -22,28 +24,30 @@ import { PageTitle } from '~/components/page-title';
 import { Progress } from '~/components/progress';
 import { AppError } from '~/errors/app-error';
 import { ErrorCodes } from '~/errors/error-codes';
-import { getFixedT, getTranslation } from '~/i18n-config.server';
+import { getTranslation } from '~/i18n-config.server';
 import { handle as parentHandle } from '~/routes/protected/layout';
 import { REGEX_PATTERNS } from '~/utils/regex-utils';
 import { trimToUndefined } from '~/utils/string-utils';
 
 type BirthDetailsSessionData = NonNullable<SessionData['inPersonSINCase']['birthDetails']>;
 
 const REQUIRE_OPTIONS = { yes: 'Yes', no: 'No' } as const;
-const COUNTRIES = ['CAN', 'FRA'] as const;
 
 export const handle = {
   i18nNamespace: [...parentHandle.i18nNamespace, 'protected'],
 } as const satisfies RouteHandle;
 
 export async function loader({ context, request }: Route.LoaderArgs) {
   requireAuth(context.session, new URL(request.url), ['user']);
-  const t = await getFixedT(request, handle.i18nNamespace);
+  const { lang, t } = await getTranslation(request, handle.i18nNamespace);
+  const { PP_CANADA_COUNTRY_CODE } = serverEnvironment;
```

Comment:
I think env. variable that are exposes to the client must comes from `clientEnvironment`. @gregory-j-baker what do you think?

```suggestion
  const { PP_CANADA_COUNTRY_CODE } = clientEnvironment;
```

Reply:
@gregory-j-baker we can create another PR to fix this and exposing the client env. variables throught a custom hook.

```
type AppEnvironment = typeof globalThis.__appEnvironment;

export function useAppEnv(): AppEnvironment {
  return globalThis.__appEnvironment;
}
```